hedera-node/src/test/java/com/hedera/services/txns/schedule/ScheduleCreateTransitionLogicTest.java

```diff
@@ -145,6 +183,252 @@ public void followsHappyPath() {
         verify(txnCtx).setStatus(SUCCESS);
     }
 
+    @Test
+    public void validNotReadyForExecutionOtherPartyKeyNotFound() {
+        // given:
+        givenValidTxnCtx();
+        // and:
+        given(store.getScheduleID(transactionBody, payer)).willReturn(EMPTY_SCHEDULE);
+        given(store.createProvisionally(
+                eq(transactionBody),
+                eq(payer),
+                eq(payer),
+                eq(RichInstant.fromJava(now)),
+                argThat(jKey -> true)))
+                .willReturn(CreationResult.success(schedule));
+        // and:
+        given(store.addSigners(
+                eq(schedule),
+                argThat(jKeySet -> true))).willReturn(OK);
```

Comment:
This pice of code is copied in a lot of places. Maybe we can include it inside `givenValidTxnCtx` or add new function that calls:
`givenValidTxnCtx` + the following 3 given lines (getScheduleID, createProvisionally and addSigners)